_posts/2024-06-06-opensearch-partnering-with-datastax-on-generative-ai.md

```diff
@@ -0,0 +1,48 @@
+
+To power the explosive growth within the Generative AI space, we need to keep innovating on the tooling available to developers. These tools require access to a variety of enterprise data and we want to be there to provide that access in whatever common format is required. Being able to retrieve data in the most flexible ways possible is a necessary catalyst for getting RAG and GenAI knowledge applications to production. 
+
+Amazon sponsors the OpenSearch project to ensure the continuing existence of an open source search engine that users can use, modify, and extend however they wish.  In addition to AWS, the OpenSearch community is full of active contributors, maintainers, and partners. For Generative AI specifically, OpenSearch brings:
+
+* **Ease of use**: OpenSearch provides easy-to-use indexing and search capabilities and has built in features for text analysis, tokenization, and relevance scoring
+* **Optimized for text retrieval**: OpenSearch makes it easy to find and rank documents based on keyword queries
+* **Versatility**: OpenSearch can handle a wide variety of data types and formats
+* **AI/ML integration**: OpenSearch supports semantic search with vector embeddings, multi-modal search, hybrid search with score normalization, and sparse vector search
+
+DataStax is a leading contributor to a range of open source projects, including [Langflow](https://langflow.org/), [Apache Cassandra](https://cassandra.apache.org/_/index.html), and [JVector](https://github.com/jbellis/jvector), which provides vector search through DiskANN and advanced GenAI techniques like COLBert. Generative AI developers seek this database and vector combination to provide: 
+
+* **Context assembly**: Langflow delivers a UI to discover ecosystem components and compose the workflows that back Generative AI applications
+* **Similarity search**: JVector offers high-performance vector similarity search and can handle embedding-based queries which require low latency and high relevance
+* **Scalability**: Cassandra offers scalable persistence for structured and semi-structured data
+
+The combination of these technologies enable semantic and keyword searches as well as hybrid query processing. Context is assembled using: 
+* Keyword queries which are directed to OpenSearch to retrieve relevant documents
+* Semantic queries use JVector and Cassandra to find the most relevant data points based on vector similarity
+* Database queries which provide known personalization, profile, and transactional data
+
+### **Moving Forward**
+DataStax will maintain a JVector integration for OpenSearch, and offer OpenSearch as part of their self-managed offering platform, HCDP (Hyper Converged Data Platform) and as integration to their cloud service, Astra. 
```

Comment:
Should we include links to information about HCDP and Astra?

Reply:
We have call to action on bottom that lands pretty close.